ui/src/components/_Overview.tsx

```diff
@@ -1,60 +1,16 @@
 import { Bar, BarChart, ResponsiveContainer, XAxis, YAxis } from 'recharts'
+import { ValidatorDetailsProps } from '@/components/ValidatorDetails'
```

Comment:
Instead of importing `ValidatorDetailsProps` from `ValidatorDetails`, just create a new `OverviewProps` interface. Then if either component's props change, it won't break anything.

Reply:
You can also rename this to `Overview` (no underscore)... I did that because it was a "placeholder" component